src/command-line/index.js

```diff
@@ -15,6 +15,19 @@ program.version(Helper.getVersion(), "-v, --version")
 
 Helper.setHome(program.home || process.env.LOUNGE_HOME);
 
+require("./start");
+require("./config");
+require("./list");
+require("./add");
+require("./remove");
+require("./reset");
+require("./edit");
+
+if (!process.argv.slice(2).length || process.argv.includes("--help")) {
```

Comment:
I can't find a way to use commander to do this. You have to parse the program to get access to program.args, but as soon as you parse, you run the commands. So, sadly we'll have to stick with this. But it works.